app/Http/Controllers/UserController.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+
+class UserController extends Controller
+{
+    //
+}
```

Comment:
is this controller for the future profile view or user editing right ?

Reply:
for future